generatebundlefile/ecr_helper.go

```diff
@@ -73,13 +73,19 @@ func splitECRName(s string) (string, string, error) {
 		if len(chartNameList) == 4 {
 			return fmt.Sprintf("%s/%s", chartNameList[2], chartNameList[3]), chartNameList[3], nil
 		}
+		if len(chartNameList) > 4 {
+			return strings.Join(chartNameList[2:], "/"), chartNameList[len(chartNameList)-1], nil
+		}
 	}
 	if len(chartNameList) == 2 {
 		return chartNameList[1], chartNameList[1], nil
 	}
 	if len(chartNameList) == 3 {
```

Comment:
Same comment here for ==3 and >3 scenario. Can they be consolidated?

Reply:
Yea lemme test that out

Reply:
ok pushed with c930afcdf3ffeaafdbfc746c124542c2067710b3